githubbot/githubbot/http.go

```diff
@@ -136,10 +137,16 @@ func (h *HTTPSrv) handleWebhook(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *HTTPSrv) formatMessage(event interface{}, repo string, defaultBranch string, client *github.Client) (message string, branch string) {
+	parsedRepo := strings.Split(repo, "/")
+	if len(parsedRepo) != 2 {
+		h.Debug("invalid repo: %s", repo)
+		return
+	}
+
 	branch = defaultBranch
 	switch event := event.(type) {
 	case *github.IssuesEvent:
-		author := getPossibleKBUser(h.kbc, h.db, h.DebugOutput, event.GetSender().GetLogin())
+		author := getPossibleKBUser(h.kbc, h.db, h.DebugOutput, event.GetIssue().GetUser().GetLogin())
 		message = git.FormatIssueMsg(
```

Comment:
preexisting but can these jsut be `return` directly? i think it helps with readability